### DISCUSSION TITLE
Create new folder as document name

### DISCUSSION THREAD

**Discussion by JoshWaldner** [TimPaterson/Fusion360-Batch-Post#62] [Ideas]
is it possible to drop the g-code files into a new folder created automatically with the name of the document as the name of the folder?

**Comment by JoshWaldner** [comment]
i found a quick fix of what i am trying to do here.
i changed line 703 to this:

703      `outputFolder = docSettings["output"] + "/" + doc.name`

**Comment by TimPaterson** [comment]
I guess I don't understand why you wouldn't just use the `PostProcessAll` dialog to set the output folder name to what you want.

**Reply by JoshWaldner** [reply]
Mass production. Me and my coworkers go through several documents per day, and post process each nest separately into its own folder so that cnc operators can keep each nest/job separate. This postprocessall addin works perfect for our application. The dialog is set to a shared folder in which we post all the nests separately. It is like instead writing job number each time ("Y:\Holzher\55-23-4657"), the job number will automatically be a folder within "Y:\Holzher"
